CMakeLists.txt

```diff
@@ -601,6 +601,7 @@ set(COVERAGE_ARGUMENTS
 
 if(FIPS)
     set(TEST_FIPS_PROPERTY "-DFIPS=true")
+    set(REGISTER_RNG_PROPERTY "-Dcom.amazon.corretto.crypto.provider.registerSecureRandom=true")
```

Comment:
This warrants a comment to explain the rationale as it does run counter to the default behavior of the library. Something along the lines of "we want to test our own deterministically selected RNG in FIPS mode".

Reply:
good call. will add.